NEWS

```diff
@@ -1,3 +1,31 @@
+11-May-2022 Ignition 2.14.0
+
+    Features:
+
+    - Support KubeVirt platform
+    - Support AWS arn: URLs for S3 objects and access points
+    - Support reading configs from Azure IMDS "user data"
+    - Support S3 fetch via IPv6
+    - Add ignition-apply entrypoint to apply an Ignition config in a container
+
+  Changes:
+
+    - Delete userdata after provisioning on VirtualBox and VMware by default
+      (see operator notes for details)
+    - Support setting setuid/setgid/sticky mode bits (3.4.0-exp)
+    - Warn if setuid/setgid/sticky mode bits specified (3.0.0 - 3.3.0)
+    - Support UEFI Secure Boot on VMware
+    - Add arm64 support to ignition-validate container
+    - Document S3 fetch semantics in operator notes
+
```

Comment:
Let's add an item for https://github.com/coreos/ignition/pull/1365, maybe something like "Document considerations for handling secrets in operator notes".  This is more general than the userdata deletion parenthetical above; I think there's value in mentioning both.

Reply:
I went with your wording to the T. Thank you for the suggestion